.config-schema.json

```diff
@@ -349,6 +349,30 @@
                     "x-env-variable": "OPENFGA_METRICS_ENABLE_RPC_HISTOGRAMS"
                 }
             }
+        },
+        "resolveCheckCache": {
```

Comment:
I know we've been back and forth, but `ResolveCheck` is a deeply internal interface. Please keep this more generic.

I suggest `--check-query-cache-enabled`, `--check-query-cache-limit`, `--check-query-cache-ttl`.